site2/docs/client-libraries-java.md

```diff
@@ -22,6 +22,11 @@ This document focuses only on the client API for producing and consuming message
 
 The latest version of the Pulsar Java client library is available via [Maven Central](http://search.maven.org/#artifactdetails%7Corg.apache.pulsar%7Cpulsar-client%7C{{pulsar:version}}%7Cjar). To use the latest version, add the `pulsar-client` library to your build configuration.
 
+> **Tip**
+>
+> - [`pulsar-client`](https://search.maven.org/artifact/org.apache.pulsar/pulsar-client) and [`pulsar-client-admin`](https://search.maven.org/artifact/org.apache.pulsar/pulsar-client-admin) shade dependencies. If you do not want to manage dependency conflicts, you can use them.
```

Comment:
The term `shade dependencies` here is too simple and might make users confused.

My suggested change is:

> ... shade dependencies via [maven-shade-plugin](https://maven.apache.org/plugins/maven-shade-plugin/) to avoid conflicts of the underlying dependency packages like Netty.

BTW, here is a blog about how the Maven shade plugin solves the dependency conflicts: https://developpaper.com/using-maven-shade-plugin-to-solve-incompatible-dependency-conflicts/